resources/smcp-templates/v2.5/base

```diff
@@ -14,7 +14,7 @@ spec:
     egress:
       enabled: true
     openshiftRoute:
-      enabled: true
+      enabled: false
```

Comment:
What happens in cases such as this:
1. A user has been using IOR using SMCP with version 2.4, without explicitly enabling IOR
2. Prior to the introduction of your new UPDATE logic, they upgrade to SMCP v2.5
3. After this change gets released and their operator updated they either...
    1. `kubectl replace` the SMCP, or
    2. `kubectl apply` some unrelated change to the SMCP

In this case I'm pretty sure `kubectl replace` would cause IOR to become disabled. I'm not sure about `kubectl apply`?